src/mscorlib/src/System/Runtime/CompilerServices/Unsafe.cs

```diff
@@ -121,6 +145,19 @@ public static bool AreSame<T>(ref T left, ref T right)
             throw new InvalidOperationException();
         }
 
+        /// <summary>
+        /// Initializes a block of memory at the given location with a given initial value 
+        /// without assuming architecture dependent alignment of the address.
+        /// </summary>
+        [NonVersionable]
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static void InitBlockUnaligned(void* startAddress, byte value, uint byteCount)
```

Comment:
This one should not be needed. As far as I can tell, it is only used by the OOB-specific code.

Reply:
As yes, you are right. It is only used by Fill and Clear which have different implementations for OOB/inbox.

Should we try to keep the set of Unsafe APIs in mscorlib at a minimum? Is there an issue with having unused APIs or should I remove them?

Reply:
I would only keep the ones that are actually used.